SingleSource/UnitTests/AArch64/CMakeLists.txt

```diff
@@ -1,10 +1,9 @@
 if(CMAKE_C_COMPILER_ID STREQUAL "Clang")
+  if(TARGET_OS STREQUAL "Linux")
+    list(APPEND CFLAGS --rtlib=compiler-rt)
+    list(APPEND LDFLAGS --rtlib=compiler-rt)
+  endif()
   if(ARCH STREQUAL "AArch64")
-    set(Source )
-    # TODO: this test is currently only supported on Darwin platforms:
-    if (TARGET_OS MATCHES "Darwin")
-      list(APPEND Source acle-fmv-features.c)
-    endif()
```

Comment:
Does it get added to `Sources` automatically for all platforms now?

Reply:
I think so looking into https://github.com/llvm/llvm-test-suite/blob/458fed38c2fb9db63b6dd8ced9708e557f3657ff/SingleSource/UnitTests/CMakeLists.txt#L11
But I tested that on aarch64/x86 Linux only

Reply:
ok, seems fine. If we have trouble with non-posix systems (Fuchsia? Windows?) I guess those can be opted out if the buildbots complain.